vnext/codegen/rnwcoreJSI-generated.cpp

```diff
@@ -1101,6 +1101,12 @@ static jsi::Value __hostFunction_NativeDevSettingsCxxSpecJSI_addMenuItem(jsi::Ru
   );
   return jsi::Value::undefined();
 }
+static jsi::Value __hostFunction_NativeDevSettingsCxxSpecJSI_openDebugger(jsi::Runtime &rt, TurboModule &turboModule, const jsi::Value* args, size_t count) {
+  static_cast<NativeDevSettingsCxxSpecJSI *>(&turboModule)->openDebugger(
+    rt
+  );
+  return jsi::Value::undefined();
```

Comment:
Same as above
```
  // TODO: <implement openDebugger issue number>
  return jsi::Value::undefined();
```

Reply:
Reverting this comment as it is a generated file